Dockerfile

```diff
@@ -41,6 +41,11 @@ RUN npm run build
 # Finally, build the production image with minimal footprint
 FROM base
 
+ENV DATABASE_URL=file:/data/sqlite.db
+
+# add shortcut for connecting to database CLI
```

Comment:
Could you add the note about how to connect to this in production with the fly console command?